src/SessionSetup/TargetLabelTest.cpp

```diff
@@ -36,17 +38,44 @@ TEST(TargetLabel, ChangeToFileTarget) {
   EXPECT_EQ(label.GetFilePath().value(), path);
 }
 
+const char* kProcessName = "test process";
+const char* kInstanceName = "test instance";
+const double kCpuUsage = 50.1;
+const char* kCpuUsageDisplay = "50";
+
+void ChangeToFakeStadiaTarget(TargetLabel& label) {
+  orbit_client_data::ProcessData process;
+  orbit_grpc_protos::ProcessInfo process_info;
+  process_info.set_name(kProcessName);
+  process_info.set_full_path("/mnt/developer/test_process");
+  process_info.set_cpu_usage(kCpuUsage);
+  process.SetProcessInfo(process_info);
+
+  auto maybe_instance =
+      orbit_ggp::Instance::CreateFromJson(QString{"{"
+                                                  "displayName: \"%1\", "
+                                                  "id: \"edge/test/instance\", "
+                                                  "ipAddress: \"127.0.0.1\", "
+                                                  "state: \"IN_USE\", "
+                                                  "owner: \"unit@test\", "
+                                                  "lastUpdated: \"2022-07-04T13:22:04Z\", "
+                                                  "pool: \"unit-test-pool\""
+                                                  "}"}
+                                              .arg(kInstanceName)
+                                              .toUtf8());
+
+  assert(maybe_instance.has_value());
```

Comment:
this should be an ASSERT_TRUE

Reply:
Or even better:
`EXPECT_THAT(maybe_instance, orbit_test_utils::HasValue());`
Then you get a better error message in the tests